_posts/release/2020-12-09-accumulo-1.10.1.md

```diff
@@ -33,22 +33,32 @@ own, in order to get the best response.
 
 ## Major Bug Fixes
 
-### TODO title bug fix
+* Update exceptions when permission checks fail (PR [#1830] - backport of [#1828]).
 
-## Other Miscellaneous Bug Fixes
+### Other Bug Fixes
+
+* Improve log message in Delete Cleanup FATE.
 
-* {% ghi 1644 %} TODO Describe bugs
+## Other Miscellaneous Bug Fixes
 
+* Testing and build improvements.
+* Support building native libraries on alpine-based distros.
+* Enforce build requirements for java to be at least java 11 and strictly less than java 15.
+* Update tool.sh script used for MR examples to support hadoop 2.0 and 3.0 and ZooKeeper 3.4, 3.5
+* Update release candidate script
```

Comment:
Some of these are internal development stuff, and not worth highlighting. Removing the ones that don't need to be there, especially those that are very vague, can bring more attention to the ones that users should pay attention to:

```suggestion
* Support building native libraries on alpine-based distros.
* Enforce build requirements for java to be at least java 11 and strictly less than java 15 (JDK 15 breaks, because it drops support for the CMS garbage collector that 1.10 still supports/assumes by default, especially in minicluster).
* Update tool.sh script used for MR examples to support hadoop 2.0 and 3.0 and ZooKeeper 3.4, 3.5
```